src/node.cc

```diff
@@ -981,6 +983,42 @@ void SetupNextTick(const FunctionCallbackInfo<Value>& args) {
       FIXED_ONE_BYTE_STRING(args.GetIsolate(), "_setupNextTick"));
 }
 
+void PromiseRejectCallback(PromiseRejectMessage message) {
+  Isolate* isolate = Isolate::GetCurrent();
```

Comment:
I see now that @vkurchatkin said exactly the same thing. :-)

Yes, `Isolate::GetCurrent()` is fine for now but like @trevnorris said, it's probably going to go away someday.  All other things being equal, we might as well start using `Promise::GetIsolate()` right away.